### PR TITLE
Load content into DB during initialization

### LINKED ISSUE #157: Processing 'content' for insights rules
As developer I need to be able to process the whole repository with 'content' for insights rules so I'll be able to create in-memory representation of such data to be available via REST API.

Repository with 'content' (templates for messages) for insights rules has specified structure. We need to be able to go through cloned repository and create object from data that will be stored in memory (like a cache). In case of any error, the old version of the object will be used instead so the service won't be downgraded.

See also #156

### PR DESCRIPTION
**Pull request by natiiix**
# Description

We need content and rule metadata in the database. Therefore we must parse it and load it into the DB before doing anything else.

### TODO

- [x] Add/Modify Unit Tests

Fixes #157 

## Type of change

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Testing steps

Aggregator should start successfully. This requires the `/rules-content/` directory to exist.

### CODE REVIEW

File: aggregator.go [hunk 1]
```diff
@@ -142,6 +174,12 @@ func startService() int {
 	var waitGroup sync.WaitGroup
 	exitCode := 0
 
+	prepDbExitCode := prepareDB()
+	if prepDbExitCode != 0 {
+		fmt.Printf(consumerExitedErrorMessage, prepDbExitCode)
```
Comment (bond95) [inline aggregator.go]: Please, use `fmt.Errorf` to be consistent

File: migration/migration2.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ var mig2 = Migration{
 		_, err := tx.Exec(`
 			CREATE TABLE rule (
 				module          VARCHAR PRIMARY KEY,
-				name            VARCHAR NOT NULL,
+				"name"            VARCHAR NOT NULL,
```
Comment (bond95) [inline migration/migration2.go]: Is this necessary to have here quotes? If yes, maybe will be better to add quotes to other filed names as well? WDYT?

Comment (tisnik) [reply migration/migration2.go]: imho in this case it is indeed needed

Comment (JoseLSegura) [reply migration/migration2.go]: Why is it needed? If it is, I agree with @bond95 to put quotes to every column name

Comment (natiiix) [reply migration/migration2.go]: @bond If none of the others are, `name` is a reserved keyword in PostgreSQL. Some of the others may be too, but probably not in this context.

Comment (natiiix) [reply migration/migration2.go]: @JoseLSegura We should be probably doing that always everywhere anyways. It's just a bit of a nuisance.

Comment (natiiix) [reply migration/migration2.go]: I was getting strange errors (`no column "name" in relation "rule"` or something like that) before I added the quotes.

[The documentation is here.](https://www.postgresql.org/docs/8.1/sql-keywords-appendix.html)

Comment (JoseLSegura) [reply migration/migration2.go]: It's ok for me then. TY

**Pull request merged**